R/tte_source.R

```diff
@@ -0,0 +1,256 @@
+#' Pre-Defined Time-to-Event Source Objects
+#'
+#' These pre-defined `tte_source` objects can be used as input to `derive_param_tte()`.
+#'
+#' @details
+#' To see the definition of the various objects simply print the object in the
+#' R console, e.g. `print(death_event)`.
+#'
+#' @seealso [derive_param_tte], [tte_source], [event_source], [censor_source]
```

Comment:
Updated derive_param_tte to admiral::derive_param_tte, event_source to admiral::event_source, and censor_source to admiral::censor_source. Kept tte_source as is since it's an object and the same file, if I actually need to update, please let me know.